README.md

```diff
@@ -14,19 +13,6 @@ Install from NPM:
 npm install ifrau
 ```
 
-Or include it in your application as UMD/CommonJs from the Brightspace CDN:
-
-```html
-<!-- probably what you're looking for -->
-<script src="https://s.brightspace.com/lib/ifrau/{version}/ifrau/client.js"></script>
-
-<!-- actually hosting a FRA on your page? grab the host -->
-<script src="https://s.brightspace.com/lib/ifrau/{version}/ifrau/host.js"></script>
-
-<!-- the old bundle is still available too -->
-<script src="https://s.brightspace.com/lib/ifrau/{version}/ifrau.js"></script>
-```
-
 ## Host and Client
 
```

Comment:
Should these instructions below change to not use `require()`? i.e. `import { Client, Host } from 'ifrau'`

Reply:
Ah yes, good call!